TeX_files/chapter04.tex

```diff
@@ -42,70 +41,70 @@ \chapter{Transforming our Communities}
-		\item Raise awareness: talk to friends
-		\item Be outspoken
-		\item Confront ‘isms’ as they arise
-		\item Be a good ally
-		\item Help show your community that we are here
-		\item Teach children that homeless people have feelings and that LGBT parents are just like theirs
-		\item People could educate themselves about PTSD and understand that it’s a part of who I am, and they’ll appreciate me, and other sufferers, all the more for it
-		\item Prevent bullying
-		\item ake actions together and plan actions in groups. They could educate themselves on oppression, privilege, racism, cultural competency, and so forth
-		\item Support families in your communities. Make establishments intergenerational to include children and the elderly
-		\item Demand that the homeless be allowed to sleep in their homeless camps.They could build a support system for activists and  fighting oppression and encourage their efforts and their voice and their struggle
-		\item I believe the first step is to cultivate an open mind, to believe that we are all capable of healing and change
-		\item Participate in letter-writing campaigns to companies and complain
-		\item Accept that there are people like me and that I desire to help them
-		\item Accept that I am able to do art and poetry because of my life’s journey
-		\item Go to rallies and marches to support others
-		\item Sign petitions and email politicians
-		\item Display anti-racist stickers
-		\item Talk to others to raise awareness
-		\item Create an inner peace and that is indestructible
+		\item Wykonaj pracę. Kształć się. Ludzie muszą się uczyć, mówić głośniej, być w porządku z uczuciem niewygody
+		\item Dołącz do działań na rzecz społeczeństwa. Przyczyniaj się we własny sposób, niezależnie czy będzie to maszerowanie na ulicach, czy wypełnianie kopert
+		\item Podpisuj petycje
+		\item Przemawiaj na spotkaniach Rady Miasta
+		\item Edukuj swoje dzieci
+		\item Zakończ ciszę. Mów o tym. Nazywaj to. Okaż wsparcie
+		\item Zwiększaj świadomość: rozmawiaj z przyjaciółmi
+		\item Bądź szczery
+		\item Mierz się z ‘izmami’ gdy się pojawiają
+		\item Bądź dobrym sprzymierzeńcem
+		\item Pomagaj pokazywać swojej społeczności, że jesteśmy tutaj
+		\item Ucz dzieci, że bezdomni mają uczucia, oraz że rodzice LGBT są tacy sami jak inni rodzice
+		\item Ludzie mogliby uczyć się o PTSD oraz rozumieć, że jest to częścią tego, kim jestem, docenialiby mnie oraz innych cierpiących
+		\item Zapobiegaj znęcaniu się
+		\item Działajcie razem oraz planujcie działanie w grupach. Grupy mogą kształcić się na temat opresji, przywilejów, rasizmu, kompetencji kulturowych i innych
+		\item Wspieraj rodziny w swojej społeczności. Twórz międzypokoleniowe instytucje, uwzględniając dzieci i starszych
+		\item Żądaj by bezdomni mieli pozowlenie, by spać w przytułkach. Mogliby budować system wsparcia dla aktywistów, by walczyć z opresją i wspierać ich wysiłki, ich głos oraz zmagania
```

Comment:
"Mogliby budować system" - to jest trochę dziwne, bo w sumie nie wiadomo, kto mógłby budować. Zresztą tak jak w oryginale.